### PR TITLE
Rename metric to etl_task_total

### PR DESCRIPTION
**Pull request by cristinaleonr**
Rename the `etl_task_count` metric to `etl_task_total` to honor prometheus best practices for metric naming: https://prometheus.io/docs/practices/naming/.

This change is [Reviewable](https://reviewable.io/reviews/m-lab/etl/1040)

### CODE REVIEW

**Review by cristinaleonr — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/m-lab/etl/1040)* status: 0 of 1 approvals obtained

___
*[cmd/update-schema/update.go, line 251 at r2](https://reviewable.io/reviews/m-lab/etl/1040#-MrU8YtAEz1u6a4ano5A:-MrU8YtAEz1u6a4ano5B:b1dpxb8) ([raw file](https://github.com/m-lab/etl/blob/9cd6380f6b7c0a3431fe14a024b25248dfc91297/cmd/update-schema/update.go#L251)):*
> ```Go
> 	}
> 
> 	// if err := CreateOrUpdatePCAPRow(project, "tmp_ndt", "pcap"); err != nil {
> ```

This needs to be uncommented before merge.

**Review by stephen-soltesz — COMMENTED** [review]
The PR is still draft, so just to comment that this looks right to me. I would recommend `grep`ing promethes-support for dashboards that contain `etl_task_count`. And, I think that may be it.

> *[Reviewable](https://reviewable.io/reviews/m-lab/etl/1040#-:-MrX_O97DcvAzSLAoDVM:b7s4rxy)* status: 0 of 1 approvals obtained

**Review by stephen-soltesz — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/m-lab/etl/1040)* status: 0 of 1 approvals obtained (waiting on @cristinaleonr)

___
*[metrics/metrics.go, line 189 at r2](https://reviewable.io/reviews/m-lab/etl/1040#-MrXaR12E7H-lAcpte7Q:-MrXaR12E7H-lAcpte7R:bdy9vnh) ([raw file](https://github.com/m-lab/etl/blob/9cd6380f6b7c0a3431fe14a024b25248dfc91297/metrics/metrics.go#L189)):*
> ```Go
> 	// Example usage:
> 	//   metrics.TaskTotal.WithLabelValues("ndt", "Task", "ok").Inc()
> 	TaskTotal = promauto.NewCounterVec(
> ```

Hmm, Should we call these variables `FooCount` or `FooTotal`? There are lots of existing examples of `FooCount`. And, this question came up in https://reviewable.io/reviews/m-lab/etl-gardener/355 when I noticed that it's using `FooCount`. I don't have a strong opinion other than we should pick one way and do that consistently. The smallest edit distance would be to use `TaskCount` here. I can make a more satisfying argument for `FooTotal` but that's not for this PR.

**Review by cristinaleonr — COMMENTED** [review]
The reason why I made it a draft is because there is a conflict between the current PCAP schema in Sandbox and the one in master, so that needs to be resolved before this can be merged.

I changed all the `etl_task_count` references to `etl_task_total` in prometheus-support in https://github.com/m-lab/prometheus-support/pull/858.

> *[Reviewable](https://reviewable.io/reviews/m-lab/etl/1040#-:-MrYbkbyCHn_3cvLXO5Z:b-z2pryt)* status: 0 of 1 approvals obtained (waiting on @stephen-soltesz)

___
*[metrics/metrics.go, line 189 at r2](https://reviewable.io/reviews/m-lab/etl/1040#-MrXaR12E7H-lAcpte7Q:-MrYdaSYBtKKqR-4_AO5:blsmps5) ([raw file](https://github.com/m-lab/etl/blob/9cd6380f6b7c0a3431fe14a024b25248dfc91297/metrics/metrics.go#L189)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Hmm, Should we call these variables `FooCount` or `FooTotal`? There are lots of existing examples of `FooCount`. And, this question came up in https://reviewable.io/reviews/m-lab/etl-gardener/355 when I noticed that it's using `FooCount`. I don't have a strong opinion other than we should pick one way and do that consistently. The smallest edit distance would be to use `TaskCount` here. I can make a more satisfying argument for `FooTotal` but that's not for this PR.
</blockquote></details>

As discussed, `FooTotal` follows Prometheus best practices for metric naming. I made an issue to change the others, too https://github.com/m-lab/etl/issues/1041.

**Review by stephen-soltesz — APPROVED** [review]
Ah, okay. :lgtm: once the pcap changes are unnecessary.

> *[Reviewable](https://reviewable.io/reviews/m-lab/etl/1040#-:-MrZGiL0-85V7G0j9ps7:b-v1dktd)* status: :shipit: complete! 1 of 1 approvals obtained

**Pull request merged**
